### ISSUE TITLE
"Process" keyword is incorrectly highlighted in New-Object -TypeName System.Diagnostics.Process

### ISSUE THREAD

**Issue by pcgeek86** [PowerShell/EditorSyntax#47]
```
New-Object -TypeName System.Diagnostics.Process -Property @{  }
```

<img width="559" alt="screen shot 2017-04-23 at 10 28 32 am" src="https://cloud.githubusercontent.com/assets/466713/25315775/9e8feb84-280f-11e7-8b8f-22a9378a2bf9.png">

Cheers,
Trevor Sullivan

**Comment by omniomi** [comment]
Need to add `.` to the negative lookbehind for `keyword.control.powershell`

**Comment by omniomi** [comment]
This was fixed by #101
